{
    "timestamp": "12/18/2024 6:08:09",
    "title": "CrustyBase v.2.0: new features and enhanced utilities to support open science",
    "link_to_resource": "https://doi.org/10.1186/s12864-024-10033-7",
    "creators": [
        "Camerion J Hyde",
        "Tomer Ventura"
    ],
    "material_type": [
        "Reading"
    ],
    "education_level": [
        "College / Upper Division (Undergraduates)",
        "Graduate / Professional"
    ],
    "abstract": "Background\nTranscriptomes present a rich, multi-dimensional subset of genomics data. They provide broad insights into genetic sequence, and more significantly gene expression, across biological samples. This technology is frequently employed for describing the genetic response to experimental conditions and has created vast libraries of datasets which shed light on gene function across different tissues, diseases, diets and developmental stages in many species. However, public accessibility of these data is impeded by a lack of suitable software interfaces and databases with which to locate and analyse them.\n\nBody\nHere we present an update on the status of CrustyBase.org, an online resource for analysing and sharing crustacean transcriptome datasets. Since its release in October 2020, the resource has provided many thousands of transcriptome sequences and expression profiles to its users and received 19 new dataset imports from researchers across the globe. In this article we discuss user analytics which point towards the utilization of this resource. The architecture of the application has proven robust with over 99.5% uptime and effective reporting of bugs through both user engagement and the error logging mechanism.\n\nWe also introduce several new features that have been developed as part of a new release of CrustyBase.org. Two significant features are described in detail, which allow users to navigate through transcripts directly by submission of transcript identifiers, and then more broadly by searching for encoded protein domains by keyword. The latter is a novel and experimental feature, and grants users the ability to curate gene families from any dataset hosted on CrustyBase in a matter of minutes. We present case studies to demonstrate the utility of these features.\n\nConclusion\nCommunity engagement with this resource has been very positive, and we hope that improvements to the service will further enable the research of users of the platform. Web-based platforms such as CrustyBase have many potential applications across life science domains, including the health sector, which are yet to be realised. This leads to a wider discussion around the role of web-based resources in facilitating an open and collaborative research community.",
    "language": [
        "English"
    ],
    "conditions_of_use": "CC BY",
    "primary_user": [
        "Student",
        "Teacher"
    ],
    "subject_areas": [
        "Life Science"
    ],
    "FORRT_clusters": [
        "Reproducible Analyses",
        "Open Data and Materials"
    ],
    "tags": [
        "Crustacea",
        "Differential Gene Expression",
        "Online Resource",
        "RNA-Seq",
        "Transcriptome"
    ]
}