{
    "timestamp": "9/18/2023 8:43:45",
    "title": "Guidelines for Evaluating the Comparability of Down-Sampled GWAS Summary Statistics",
    "link_to_resource": "https://doi.org/10.1007/s10519-023-10152-z",
    "creators": [
        "Camille M. Williams",
        "Holly Poore",
        "Peter T. Tanksley",
        "Hyeokmoon Kweon",
        "Natasia S. Courchesne-Krak",
        "Diego Londono-Correa",
        "Travis T. Mallard",
        "Peter Barr",
        "Philipp D. Koellinger",
        "Irwin D. Waldman",
        "Sandra Sanchez-Roige",
        "K. Paige Harden",
        "Abraham A. Palmer",
        "Danielle M. Dick & Richard Karlsson Linn\u00e9r"
    ],
    "material_type": [
        "Reading"
    ],
    "education_level": [
        "College / Upper Division (Undergraduates)",
        "Graduate / Professional",
        "Career /Technical"
    ],
    "abstract": "Proprietary genetic datasets are valuable for boosting the statistical power of genome-wide association studies (GWASs), but their use can restrict investigators from publicly sharing the resulting summary statistics. Although researchers can resort to sharing down-sampled versions that exclude restricted data, down-sampling reduces power and might change the genetic etiology of the phenotype being studied. These problems are further complicated when using multivariate GWAS methods, such as genomic structural equation modeling (Genomic SEM), that model genetic correlations across multiple traits. Here, we propose a systematic approach to assess the comparability of GWAS summary statistics that include versus exclude restricted data. Illustrating this approach with a multivariate GWAS of an externalizing factor, we assessed the impact of down-sampling on (1) the strength of the genetic signal in univariate GWASs, (2) the factor loadings and model fit in multivariate Genomic SEM, (3) the strength of the genetic signal at the factor level, (4) insights from gene-property analyses, (5) the pattern of genetic correlations with other traits, and (6) polygenic score analyses in independent samples. For the externalizing GWAS, although down-sampling resulted in a loss of genetic signal and fewer genome-wide significant loci; the factor loadings and model fit, gene-property analyses, genetic correlations, and polygenic score analyses were found robust. Given the importance of data sharing for the advancement of open science, we recommend that investigators who generate and share down-sampled summary statistics report these analyses as accompanying documentation to support other researchers\u2019 use of the summary statistics.",
    "language": [
        "English"
    ],
    "conditions_of_use": "CC BY",
    "primary_user": [
        "Student",
        "Teacher",
        "researcher"
    ],
    "subject_areas": [
        "Life Science"
    ],
    "FORRT_clusters": [
        "Conceptual and Statistical Knowledge",
        "Reproducible Analyses"
    ],
    "tags": [
        "Genomic SEM",
        "Summary Statistics",
        "Data Removal",
        "Down-sample",
        "Leave-one-out",
        "Meta-analysis",
        "Genomics",
        "Genome-wide Association Study"
    ]
}