{
    "timestamp": "01/01/2020",
    "title": "Intro to R and RStudio for Genomics",
    "link_to_resource": "https://datacarpentry.org/genomics-r-intro/",
    "creators": [
        "Ahmed Moustafa",
        "Alexia Cardona",
        "Andrea Ortiz",
        "Jason Williams",
        "Krzysztof Poterlowicz",
        "Naupaka Zimmerman",
        "Yuka Takemon"
    ],
    "material_type": [
        "Module"
    ],
    "education_level": [
        "Graduate / Professional"
    ],
    "abstract": "Welcome to R! Working with a programming language (especially if it\u2019s your first time) often feels intimidating, but the rewards outweigh any frustrations. An important secret of coding is that even experienced programmers find it difficult and frustrating at times \u2013 so if even the best feel that way, why let intimidation stop you? Given time and practice* you will soon find it easier and easier to accomplish what you want. Why learn to code? Bioinformatics \u2013 like biology \u2013 is messy. Different organisms, different systems, different conditions, all behave differently. Experiments at the bench require a variety of approaches \u2013 from tested protocols to trial-and-error. Bioinformatics is also an experimental science, otherwise we could use the same software and same parameters for every genome assembly. Learning to code opens up the full possibilities of computing, especially given that most bioinformatics tools exist only at the command line. Think of it this way: if you could only do molecular biology using a kit, you could probably accomplish a fair amount. However, if you don\u2019t understand the biochemistry of the kit, how would you troubleshoot? How would you do experiments for which there are no kits? R is one of the most widely-used and powerful programming languages in bioinformatics. R especially shines where a variety of statistical tools are required (e.g. RNA-Seq, population genomics, etc.) and in the generation of publication-quality graphs and figures. Rather than get into an R vs. Python debate (both are useful), keep in mind that many of the concepts you will learn apply to Python and other programming languages. Finally, we won\u2019t lie; R is not the easiest-to-learn programming language ever created. So, don\u2019t get discouraged! The truth is that even with the modest amount of R we will cover today, you can start using some sophisticated R software packages, and have a general sense of how to interpret an R script. Get through these lessons, and you are on your way to being an accomplished R user! * We very intentionally used the word practice. One of the other \u201csecrets\u201d of programming is that you can only learn so much by reading about it. Do the exercises in class, re-do them on your own, and then work on your own problems.",
    "language": [
        "English"
    ],
    "conditions_of_use": "Creative Commons Attribution 4.0 International",
    "primary_user": [
        "student",
        "teacher"
    ],
    "subject_areas": [
        "Computer Science",
        "Information Science",
        "Biology",
        "Measurement and Data"
    ],
    "FORRT_clusters": [
        ""
    ],
    "tags": [
        "Analysis",
        "Data",
        "Education",
        "Inside Your Classroom",
        "Open Scholarship Tools and Technologies",
        "R",
        "Reproducibility",
        "Research Data Management Tools",
        "Researchers"
    ]
}